DependencyInjection/Configuration.php

```diff
@@ -305,9 +305,20 @@ private function addConnectionsSection(ArrayNodeDefinition $rootNode)
                                     ->end()
                                     ->integerNode('socketTimeoutMS')->end()
                                     ->booleanNode('ssl')->end()
+                                    ->booleanNode('tls')->end()
+                                    ->booleanNode('tlsAllowInvalidCertificates')->end()
+                                    ->booleanNode('tlsAllowInvalidHostnames')->end()
+                                    ->scalarNode('tlsCAFile')->end()
+                                    ->scalarNode('tlsCertificateKeyFile')->end()
+                                    ->scalarNode('tlsCertificateKeyFilePassword')->end()
+                                    ->booleanNode('tlsDisableCertificateRevocationCheck')->end()
+                                    ->booleanNode('tlsDisableOCSPEndpointCheck')->end()
+                                    ->booleanNode('tlsInsecure')->end()
                                     ->scalarNode('username')
                                         ->validate()->ifNull()->thenUnset()->end()
                                     ->end()
+                                    ->scalarNode('retryReads')->end()
+                                    ->scalarNode('retryWrites')->end()
```

Comment:
Based on the docs and usage:
```suggestion
                                    ->booleanNode('retryReads')->end()
                                    ->booleanNode('retryWrites')->end()
```

Reply:
Sorry, that was sloppy. Fixed and squashed